### PR TITLE
#175 - Do not build assembly jar-with-dependencies

### PR DESCRIPTION
**Pull request by marceloamadeu**
This PR is for #175:

- Profile **ver** created in `pom.xml`

*** We can change the profile ID, if necessary.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#186](https://github.com/artipie/artipie/issues/186) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/823b0c8c-3327-4d87-837f-64aa50bf84b1); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: pom.xml [hunk 1]
```diff
@@ -198,51 +198,66 @@ SOFTWARE.
...
+      <build>
+        <testResources>
+          <testResource>
+            <directory>${basedir}/src/test/resources</directory>
+            <filtering>true</filtering>
+          </testResource>
+        </testResources>
+        <plugins>
+          <plugin>
+            <artifactId>maven-assembly-plugin</artifactId>
```
Comment (carlosmiranda) [inline pom.xml:216]: @marceloamadeu based on my interpretation of #175, we should move this to a _separate_ profile instead of including i in `qulice` profile. Besides, it does not make sense to tie this in to the static analysis of the Qulice plugin which is what the qulice profile.

Tl;dr, let's name it something other than `qulice` which is a profile we use for something else.

Comment (marceloamadeu) [reply pom.xml:216]: @carlosmiranda I understand your request for change ... Before I do I would like to ask @victornoel's opinion.

Tks

Comment (victornoel) [reply pom.xml:216]: @carlosmiranda @marceloamadeu I don't see any reason to have the maven-assembly-plugin in the same profile as qulice, those are two very different things and most certainly not meant to be used for the same reason.

I would think introducing a separate profile should be a good solution.

Note that I have no idea what this assembly file is used for, so maybe @g4s8 can chip in to tell us where the profile should be activated in the CI or in rultor for releases maybe?

Comment (victornoel) [reply pom.xml:216]: @carlosmiranda also, why put the tests related things in a profile? I think there were perfects where they were before IMHO.

Comment (g4s8) [reply pom.xml:216]: @victornoel yes, you are right, this profile should be enabled with Docker builds in Github actions.

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

File: pom.xml (hunk 1 above)
Comment (carlosmiranda) [reply pom.xml:216]: @marceloamadeu any updates on this?

**Comment by g4s8** [comment]
@marceloamadeu ping, how it's going?

**Comment by marceloamadeu** [comment]
@carlosmiranda @g4s8 @victornoel Please verify that the change in the `pom.xml` file and also in the `Dockerfile` is correct.
Tks

**Comment by victornoel** [comment]
@marceloamadeu yep, it looks good to me

**Comment by marceloamadeu** [comment]
@victornoel tks...
@carlosmiranda ping

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Code review was too long (10 days), architects (@g4s8) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by sereshqua** [comment]
@carlosmiranda please make sure you will find at least 3 issues during next CR, thanks!

**Comment by carlosmiranda** [comment]
@sereshqua confirmed, thanks

**Comment by sereshqua** [comment]
@0crat quality acceptable
